bql/semantic/expression.go

```diff
@@ -249,6 +250,44 @@ func (e *comparisonForNodeLiteral) Evaluate(r table.Row) (bool, error) {
 	}
 }
 
+// comparisonForTimeLiteral is the internal representation of an expression of comparison between a binding and a time literal.
+type comparisonForTimeLiteral struct {
+	op OP // operation.
+
+	lB  string // left binding.
+	rTL string // right time literal.
+}
+
+func (e *comparisonForTimeLiteral) Evaluate(r table.Row) (bool, error) {
+	eL, err := cellFromRow(e.lB, r)
```

Comment:
is this variable representing a "cell" ? maybe cL? I imagine "L" comes from left?

I'd suggest giving a little more information as this already starts getting complicated when we do eL.S.

Reply:
I agree with you.
Here above I was just following the standard of the file again, but this is also going to be refactored with PR #139 for the same reason of the previous comment. I am also refactoring the correspondent variable names of the other `Evaluate` methods there.

Reply:
Got it. Thanks for explaining. Please keep in mind that for small scopes, short variables are OK. But in this case we should always try to pick name that is clear e.g 'c' for counter. If in the future refactor this will still be a short variable, we could use cell, or lcell and rcell.